scalability_and_performance/ztp-deploying-disconnected.adoc

```diff
@@ -250,11 +250,11 @@ include::modules/cnf-topology-aware-lifecycle-manager-operator-update.adoc[level
 
 * For more information about updating GitOps ZTP, see xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#ztp-upgrading-gitops-ztp_ztp-deploying-disconnected[Upgrading GitOps ZTP].
 
-include::modules/cnf-topology-aware-lifecycle-manager-PAO-update.adoc[leveloffset=+2]
+include::modules/cnf-topology-aware-lifecycle-manager-operator-and-platform-update.adoc[leveloffset=+2]
+
+include::modules/cnf-topology-aware-lifecycle-manager-PAO-update.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#ztp-roll-out-the-configuration-changes_ztp-deploying-disconnected[Upgrading GitOps ZTP].
-
-include::modules/cnf-topology-aware-lifecycle-manager-operator-and-platform-update.adoc[leveloffset=+2]
+* xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#ztp-roll-out-the-configuration-changes_ztp-deploying-disconnected[Upgrading GitOps ZTP].
```

Comment:
```suggestion
* xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#ztp-roll-out-the-configuration-changes_ztp-deploying-disconnected[Upgrading GitOps ZTP]
```

Reply:
@jeana-redhat Removed the period. To answer your question, as long as I don't update the PR, it shouldn't be a problem! 
Now reading [your other comment](https://github.com/openshift/openshift-docs/pull/51474#issuecomment-1275055918), I realize someone had already mentioned this to me, probably Kathryn... I'll keep this in mind now!